### PR TITLE
feat: add color edition

### LINKED ISSUE #1301: Allow tint changes for hair eyes and skin
PR with POC https://github.com/decentraland/builder/tree/feat/add-colors

https://images.zenhubusercontent.com/162741822/8521a6f1-f609-4109-9006-e91997ad18e0/screen_recording_2021_04_15_at_22_23_11.mov

### PR DESCRIPTION
**Pull request by nicosantangelo**
Closes #1301

### CODE REVIEW

**Comment by nachomazzara** [comment]
Amazing!! 🚀 

I have only one question. Should we need to left-align the colors instead of having them centered?

Screen Shot 2021-04-21 at 09 32 25

File: src/components/ItemEditorPage/CenterPanel/CenterPanel.tsx [hunk 1]
```diff
@@ -1,18 +1,32 @@
...
 import { t } from 'decentraland-dapps/dist/modules/translation/utils'
 import ViewPort from 'components/ViewPort'
 import { AvatarAnimation, PreviewType } from 'modules/editor/types'
+import { getSkinColors, getEyeColors, getHairColors } from 'modules/editor/utils'
 import { WearableBodyShape } from 'modules/item/types'
 import { Props } from './CenterPanel.types'
 import './CenterPanel.css'
 
-export default class CenterPanel extends React.PureComponent<Props> {
+type State = {
```
Comment (nachomazzara) [inline src/components/ItemEditorPage/CenterPanel/CenterPanel.tsx:12]: Should we need to move this to `CenterPanel.types.ts`?

File: src/components/ItemEditorPage/CenterPanel/CenterPanel.tsx [hunk 2]
```diff
@@ -23,39 +37,161 @@ export default class CenterPanel extends React.PureComponent<Props> {
...
+              onChange={this.handleHairColorChange}
+            />
+          </div>
         </div>
       </div>
     )
   }
 }
+
+type AvatarColorDropdownProps = {
```
Comment (nachomazzara) [inline src/components/ItemEditorPage/CenterPanel/CenterPanel.tsx:147]: Maybe we can move this one too. WDYT?

File: src/components/ItemEditorPage/CenterPanel/CenterPanel.tsx [hunk 3]
```diff
@@ -23,39 +37,161 @@ export default class CenterPanel extends React.PureComponent<Props> {
...
+  currentTone: Color4
+  tones: Color4[]
+  label: string
+  onChange: (color: Color4) => void
+}
+export class AvatarColorDropdown extends React.PureComponent<AvatarColorDropdownProps> {
```
Comment (nachomazzara) [inline src/components/ItemEditorPage/CenterPanel/CenterPanel.tsx:153]: What do you think of moving this component to a new file?. Could be hard to find if we need to reuse it.

**Review by nachomazzara — CHANGES_REQUESTED** [review]
Feedback related to the code.

Everything looks good! :)

What do you think of using the default values [here](https://github.com/decentraland/builder/pull/1307/files#diff-b5e172feae40dc1f99b82dab23aed491db1a069b20fdf07d6e11a79917e45ac8L119)?

File: src/components/ItemEditorPage/CenterPanel/CenterPanel.tsx (hunk 1 above)
Comment (nicosantangelo) [reply src/components/ItemEditorPage/CenterPanel/CenterPanel.tsx:12]: Yes! slipped thru the cracks

File: src/components/ItemEditorPage/CenterPanel/CenterPanel.tsx (hunk 2 above)
Comment (nicosantangelo) [reply src/components/ItemEditorPage/CenterPanel/CenterPanel.tsx:147]: That you're a good reviewer

File: src/components/ItemEditorPage/CenterPanel/CenterPanel.tsx (hunk 3 above)
Comment (nicosantangelo) [reply src/components/ItemEditorPage/CenterPanel/CenterPanel.tsx:153]: Actualy skipped unintentionally, was the intention!

**Pull request merged**